src/envo/envo-edit.owl

```diff
@@ -40559,6 +40597,31 @@ AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#has
 AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_06105023> "biofouling"@en)
 SubClassOf(<http://purl.obolibrary.org/obo/ENVO_06105023> <http://purl.obolibrary.org/obo/ENVO_06105022>)
 
+# Class: <http://purl.obolibrary.org/obo/ENVO_06105024> (adhesion)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <https://en.wikipedia.org/wiki/Adhesion>) Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <https://www.biolinscientific.com/blog/what-is-adhesion>) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_06105024> "An environmental system process during which two dissimilar materials attach to one another."@en)
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000116> <http://purl.obolibrary.org/obo/ENVO_06105024> "We'll relocate this in a hierarchy of similar processes later. We'll add different types of adhesion (mechanical, chemical, dispersive, diffusive, and electrostatic) as subclasses in later revisions."@en)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/contributor> <http://purl.obolibrary.org/obo/ENVO_06105024> <https://orcid.org/0000-0002-2996-719X>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/contributor> <http://purl.obolibrary.org/obo/ENVO_06105024> <https://orcid.org/0000-0002-4366-3088>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.obolibrary.org/obo/ENVO_06105024> <http://orcid.org/0000-0002-8166-0486>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.obolibrary.org/obo/ENVO_06105024> "2021-05-12T19:18:46Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:comment <http://purl.obolibrary.org/obo/ENVO_06105024> "The attachment is usually firm, and several mechanisms (mechanical, chemical, dispersive, diffusive, and electrostatic) have been proposed.
+http://purl.obolibrary.org/obo/NCIT_C17597
+http://purl.obolibrary.org/obo/GO_0022610
+http://purl.obolibrary.org/obo/APO_0000288"@en)
```

Comment:
You'll have to describe why you're adding these IRIs here. Add a note saying that this class is related to - but distinct from - those listed here.